### ISSUE TITLE
Second execution throws "Cannot determine the module for class AppComponent"

### ISSUE THREAD

**Issue by djarekg** [UltimateAngular/aot-loader#13]
I am using the angular-dynamic-forms project and made only one change and that is because I am on Windows. That change was to the package.json start script. I changed it to...

"start": "set NODE_ENV=development && yarn run webpack"

After running "yarn start", everything executes successfully and you can navigate to localhost:3000. But if i change any file and save it, the second execution throws...

```
ERROR in Cannot determine the module for class AppComponent in C:\Development\angular\angular-dynamic-forms-master\app
\app.component.ts! Add AppComponent to the NgModule to fix it.
webpack: bundle is now VALID.
```

**Comment by Itrulia** [comment]
A VM should help. Virtualbox + Windows 10, there are a lot of tutorials online.

**Comment by rynclark** [comment]
Yeah I tried VirtualBox before but had issues trying to npm link the loader to the windows file system - I'll try it again, cheers!

**Comment by djarekg** [comment]
Any progress with supporting Windows 10?

**Comment by rynclark** [comment]
@djarekg rewriting the path resolution parts and others and will look into how other plugins normalise them for windows over the weekend. Will also setup dual boot on the mac to test it out!

**Comment by djarekg** [comment]
Any progress on this issue?

**Comment by rynclark** [comment]
Looking to getting it fixed by tomorrow

**Comment by djarekg** [comment]
That would be awesome. Starting a new project that I keep pushing back till this gets worked out.

**Comment by djarekg** [comment]
I'm sorry to be a pain and I know you guys were at the NG conference last week, but will this issue get resolved this week?

**Comment by rynclark** [comment]
@djarekg sorry for the lack of updates!

A few changes to the AoT side of things was announced at ng-conf, so I'm currently working on aligning the aot-loader with that! Will have something very soon!

**Comment by djarekg** [comment]
Thank you as always

**Comment by irowbin** [comment]
Thanks, Waiting for that... 👍

**Comment by djarekg** [comment]
Any progress with this issue? I know you mentioned other changes needing to be made for ng4 after attending the last ng conference. Can you please provide an update or eta for Windows support.
